lumen/util.py

```diff
@@ -15,9 +15,12 @@
 from jinja2 import DebugUndefined, Environment, Undefined
 from pandas.core.dtypes.dtypes import CategoricalDtype
 from panel import state
+from panel.io.cache import _hash_funcs
 
 log = getLogger(__name__)
 
+# Temporarily patch panel _hash_funcs
+_hash_funcs[dt.date] = lambda obj: f'{type(obj)}{obj}'.encode('utf-8')
```

Comment:
Why not just pin panel? Is it because `0.14.1` is needed?

Reply:
Yes, will release 0.14.1 tomorrow or Monday so this will be removed before the Lumen release but we need this for a demo today.

Reply:
Demo driven development is of course not ideal but don't have much choice in this case.

Reply:
I understand. 

I would properly open up a PR with the removal and a pin, so it is not forgotten before the next release. But maybe that is just me being too cautious.